# they use some weird template thing in pure C code to avoid
# warnings, I think, but only for intel compilers
# We remove the special case for intel and use the general one
# Ward Poelmans <wpoely86@gmail.com>
--- matplotlib-1.4.3/extern/qhull/qhull_a.h.org	2012-01-26 04:32:07.000000000 +0100
+++ matplotlib-1.4.3/extern/qhull/qhull_a.h	2014-10-01 11:49:18.000000000 +0200
@@ -102,13 +102,8 @@
 #elif defined(__MWERKS__) && defined(__INTEL__)
 #   define QHULL_OS_WIN
 #endif
-#if defined(__INTEL_COMPILER) && !defined(QHULL_OS_WIN)
-template <typename T>
-inline void qhullUnused(T &x) { (void)x; }
-#  define QHULL_UNUSED(x) qhullUnused(x);
-#else
-#  define QHULL_UNUSED(x) (void)x;
-#endif
+
+#define QHULL_UNUSED(x) (void)x;
 
 /***** -libqhull.c prototypes (alphabetical after qhull) ********************/
 
